src/LibChorus/VcsDrivers/Mercurial/HgResumeTransport.cs

```diff
@@ -810,6 +813,8 @@ private PullStatus FinishPull(string transactionId)
 
 		private PullResponse PullOneChunk(HgResumeApiParameters request)
 		{
+			using var activity = LibChorusActivitySource.Value.StartActivity();
+			activity?.TagResumableParameters("pull", request);
```

Comment:
Extra blank line after this (or remove the one from `PushOneChunk`) for consistent formatting.